app/pages/system/networking/IpPoolPage.tsx

```diff
@@ -101,32 +119,41 @@ function UtilizationBars() {
   if (ipv4.capacity === 0 && ipv6.capacity === 0n) return null
 
   return (
-    <div className="-mt-8 mb-8 flex min-w-min flex-col gap-3 lg+:flex-row">
-      {ipv4.capacity > 0 && (
-        <CapacityBar
-          icon={<IpGlobal16Icon />}
-          title="IPv4"
-          provisioned={ipv4.allocated}
-          capacity={ipv4.capacity}
-          provisionedLabel="Allocated"
-          capacityLabel="Capacity"
-          unit="IPs"
-          includeUnit={false}
-        />
-      )}
-      {ipv6.capacity > 0 && (
-        <CapacityBar
-          icon={<IpGlobal16Icon />}
-          title="IPv6"
-          provisioned={ipv6.allocated}
-          capacity={ipv6.capacity}
-          provisionedLabel="Allocated"
-          capacityLabel="Capacity"
-          unit="IPs"
-          includeUnit={false}
-        />
-      )}
-    </div>
+    <>
+      <h2 className="mb-3 pl-3 text-mono-xs text-secondary">Capacity available</h2>
+      <div className="mb-8 flex min-w-min flex-col gap-3 lg+:flex-row">
+        {ipv4.capacity > 0 ? (
+          <CapacityBar
+            icon={<IpGlobal16Icon />}
+            title="IPv4"
+            provisioned={ipv4.allocated}
+            capacity={ipv4.capacity}
+            provisionedLabel="Allocated"
+            capacityLabel="Capacity"
+            unit="IPs"
+            includeUnit={false}
+          />
+        ) : (
+          // necessary for aligning capacity bar with Properties Table
+          <div className="w-full"></div>
+        )}
```

Comment:
This has a very funny result when there are only IPv6:

<img width="1424" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/a3edfdbe-9b1d-4b22-be4c-98aaa7547350">

Reply:
Ah, good callout.

Reply:
<img width="1217" alt="Screenshot 2024-04-22 at 4 14 35 PM" src="https://github.com/oxidecomputer/console/assets/22547/1126a2ad-08c9-4ef4-9fad-869ede20cc86">